app/jobs/idd2wdl.py

```diff
@@ -118,10 +118,21 @@ def target_filename(target, index):
   command<<<
   python3 <<CODE
   import os, sys, json, contextlib, importlib, threading, logging, subprocess, traceback
-  os.environ.update(KEY_PATH_S3="{nha_cluster_ssh_key_uri}", AWS_DEFAULT_REGION="{AWS_DEFAULT_REGION}")
+  os.environ.update(
+    KEY_PATH_S3="{nha_cluster_ssh_key_uri}",
+    AWS_DEFAULT_REGION="{AWS_DEFAULT_REGION}",
+    DEPLOYMENT_ENVIRONMENT="{DEPLOYMENT_ENVIRONMENT}"
+  )
   {install_custom_idseq_dag_version}
   import idseq_dag, idseq_dag.util.s3, idseq_dag.util.count
-  logging.basicConfig(level=logging.INFO)
+
+  root_logger = logging.getLogger()
+  root_logger.setLevel(level=logging.INFO)
+  stream_handler = logging.StreamHandler()
+  stream_handler.setFormatter(idseq_dag.util.log.JsonFormatter())
+  root_logger.addHandler(stream_handler)
```

Comment:
I'm guessing you had to do this to format logs correctly for cloudwatch?

If we need to do this, can we just call `idseq_dag.util.configure_logger()` instead?

Reply:
This, is just to output all the `log_event` from pipeline step. They coming out as `None`. True that the JSON format is great for automatic parsing by cloudwatch too.
I think that `idseq_dag.util.configure_logger()` does not work since it uses a named logger from where it is called instead of receiving a logger... but willing to try again.
Maybe we can move on with this version for now given time constraints?

Reply:
Let's continue with this version, yes.